plans/http-127-0-0-1-5401-fuzzy-pizza.md

```
@@ -13,29 +14,36 @@ This is unrelated to the currently uncommitted `wrangler.jsonc` changes (those j
 reachable via the forwarded port from the host.
 
 Files to edit (same one-line change pattern in each):
+
 - `app/apex/package.json`
 - `com/apex/package.json`
 - `net/apex/package.json`
 - `org/apex/package.json`
 
 For each, change:
+
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)